src/file/format/mod.rs

```diff
@@ -81,6 +88,9 @@ lazy_static! {
         #[cfg(feature = "json5")]
         formats.insert(FileFormat::Json5, vec!["json5"]);
 
+        #[cfg(feature = "ura")]
+        formats.insert(FileFormat::Gura, vec!["ura"]);
```

Comment:
And here

Reply:
If I'm not mistaken, this section is fine as `ura` as [`.ura` is the filename extension](https://gura.netlify.app/docs/conventions#filename-extension).

Friendly ping as changes were applied since and it'd be good to have the effort merged before it becomes too stale.